Pilot/CloudFile.swift

```diff
@@ -39,10 +37,6 @@ class CloudFile: NSObject, PilotFile {
     self.size = size
   }
 
-  func setFileType(fileType: FileType) {
```

Comment:
FileType is set once for the life of the cloud file. It shouldn't be changed nor can it be empty since it's a required parameter which is why I got rid of that.